### ISSUE TITLE
Mark grid-1.0.1 as broken

### ISSUE THREAD

**Issue by ohlab** [bioconda/bioconda-recipes#10615]
I see from this link that older versions of grid is being downloaded via conda https://anaconda.org/bioconda/grid/files . Is it possible to delete older versions or at least mark them as broken?
Thanks

**Comment by bgruening** [comment]
Hi @ohlab! Why would you want to remove them? We need to keep them for reproducibility reason and by default you get the latest version. Is that not good?

**Comment by ohlab** [comment]
From this link https://anaconda.org/bioconda/grid/files, it appears version 1.0.1 is still being downloaded. At least, recently downloaded today

**Comment by bgruening** [comment]
This can be, if someone wants to have exactly this version. It makes pipelines reproducible.

**Comment by ohlab** [comment]
Not true. Simply specifying "conda install grid" pulls the older version (see attached image)
![test](https://user-images.githubusercontent.com/4581077/44482136-950d4900-a615-11e8-969a-e46082c37059.png)

**Comment by corburn** [comment]
@ohlab  commit 8527604f9b0a introduced a `samtools=0.1.19` version constraint. The older grid version was installed because it has a compatible (unspecified) samtools version constraint with the packages(s) installed in your environment.

**Comment by ohlab** [comment]
For this reason, I want older grid versions marked as broken so it doesnt get accidentally installed.

**Comment by corburn** [comment]
@ohlab for the sake of exploring alternative options, if the old recipes are not compatible with samtools>0.1.19, the run constraints of the old recipes could be updated and build numbers bumped.

**Comment by ohlab** [comment]
The latest version is the most updated without any known bug. Would want this installed by default at all times.

**Comment by ohlab** [comment]
@corburn, can you mark older versions as broken?

**Comment by corburn** [comment]
@ohlab no, I cannot.
